### ISSUE TITLE
error retrieving resource lock

### ISSUE THREAD

**Issue by sabre1041** [redhat-cop/group-sync-operator#62]
The following error occurs when deploying `v0.0.8`

```
E1231 06:31:47.702125       1 leaderelection.go:325] error retrieving resource lock group-sync-operator/085c249a.redhat.io: leases.coordination.k8s.io "085c249a.redhat.io" is forbidden: User "system:serviceaccount:group-sync-operator:default" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace "group-sync-operator"
```

**Comment by sabre1041** [comment]
This issue is being resolved by #61

**Comment by sabre1041** [comment]
Issue has been resolved
